src/core_atmosphere/Registry.xml

```diff
@@ -466,6 +466,8 @@
 			<var name="deriv_two"/>
 			<var name="defc_a"/>
 			<var name="defc_b"/>
+                        <var name="cell_gradient_coef_x"/>
+                        <var name="cell_gradient_coef_y"/>
```

Comment:
We could enclose occurrences of these new fields in
```
#ifdef MPAS_CAM_DYCORE
```
```
#endif
```
directives to prevent them from being allocated, read, and written in stand-alone MPAS-A.

Reply:
When the MPAS LES capability is available, we could remove the preprocessing directives or define a package so these would only be allocated, read, and written if we're operating as a dycore in CAM or running with LES options enabled.